.changeset/sour-dingos-taste.md

```diff
@@ -0,0 +1,7 @@
+---
+"@thirdweb-dev/wallets": major
+"@thirdweb-dev/react-native": patch
+"@thirdweb-dev/react": patch
+---
+
+[RN/Wallets/React] Migrate wallets to WCV2 and remove WCV1 support
```

Comment:
can we not do this `[RN/Wallets/React]` stuff? it already puts this changeset in every package when it gets released based on the above metadata of which packages changed how

Reply:
do it in PR titles sure but not in the changeset, unnecessary to do so